glue_jupyter/app.py

```diff
@@ -150,6 +152,12 @@ def viewers(self):
 
     def new_data_viewer(self, *args, **kwargs):
         show = kwargs.pop('show', True)
+        if isinstance(args[0], str):
+            try:
+                viewer_cls = viewer_registry.members[args[0]]['cls']
+            except KeyError:
```

Comment:
Technically this might also fail if ``['cls']`` is not found - maybe instead of try...except you could simply check ``if args[0] in viewer_registry.members``?

Reply:
Okay, I've made this a bit more robust (and added tests) in https://github.com/glue-viz/glue-jupyter/pull/307/commits/2300589b01829e58f2d1c64225835cd921cbe21b